### DISCUSSION TITLE
Privacy feature: organization only profile display name

### DISCUSSION THREAD

**Discussion by ghost** [community/community#14717] [Other Feature Feedback, Questions, & Ideas]
My employer requires github account. And wants to have my name visible publicly in my github profile.
I'd like to be able to set in custom routing a profile name by which I could be found by coworkers visible only for other org-users in the org owned repos.

Additionally on top of that it would be nice to keep the fact of being part of organization hidden publicly but visible for other same-org users.

**Comment by fr4nk3n5731n** [comment]
I agree this would be great, my employer wants us to use our full names for the display name so it's easier to find the people you work with but I don't want to loudly shout my name or the company I work for into the internet either.
But it based on the few upvotes and low engagement on this feature request, it seems that not too many people care about this sadly enough.

**Comment by Waterstraal** [comment]
+1
